doc/src/manual/calling-c-and-fortran-code.md

```diff
@@ -853,7 +867,7 @@ macro dlsym(func, lib)
     end
```

Comment:
it was collapsed and this was the closest line github let me comment on, I think you got it